source/seedsource/views.py

```diff
@@ -96,3 +96,21 @@ def post(self, request, *args, **kwargs):
         response['Content-disposition'] = 'attachment; filename=report.pdf'
 
         return response
+
+
+class RegionsFromPoint(ListAPIView):
+    queryset = Region.objects.all()
+    serializer_class = RegionSerializer
+
+    def get_queryset(self):
+        if not self.request.query_params.get('point'):
+            return self.queryset
+        else:
+            try:
+                x, y = [float(x) for x in self.request.query_params['point'].split(',')]
+            except ValueError:
+                raise ParseError()
+
+            point = Point(x, y)
+            regions = self.queryset.filter(polygons__intersects=point)
+            return sorted(list(regions), key=lambda r: point.distance(r.polygons.centroid))
```

Comment:
Sounds good. If the number of regions we're sorting against starts to grow substantially, we can revisit this.